### DISCUSSION TITLE
Quick Question

### DISCUSSION THREAD

**Discussion by Savage-Steve** [diasurgical/DevilutionX#4112] [Q&A] [answered]
_**How Do I Get This On Xbox One S?**_

**Comment by Savage-Steve** [comment]
@Savage-Steve

**Comment by StephenCWills** [comment, marked as answer]
Please refer to the following discussion.
https://github.com/diasurgical/devilutionX/discussions/3599

Note that Xbox One/Series support was merged recently. There are no releases for these platforms yet.
https://github.com/diasurgical/devilutionX/pull/3946

**Reply by qndel** [reply]
I've Heard Capitalizing Every Word Is The Fastest Way To Make Friends
![image](https://user-images.githubusercontent.com/14297035/156688145-f4ed40ab-ddd7-4033-8ace-48651f64ed41.png)
